internal/cmd/get/get.go

```diff
@@ -61,6 +62,10 @@ func getCommand(ctx context.CLIContext, opts *options, args []string) {
 		log.Fatalf("scope initialization failed: %s", err)
 	}
 
+	if _, err := semver.StrictNewVersion(opts.Version); err != nil {
+		log.Fatalf("'%s' does not meet the criteria for semantic versioning", opts.Version)
```

Comment:
heh, I know I should've mentioned it before, but you could also add the error message returned by that semver check to your fatal log <3

Reply:
Done